datarepo-clienttests/src/main/resources/configs/DRSLookup.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "DRSLookup",
+  "description": "Run DRS lookups on the file refs in two different snapshots.",
```

Comment:
Nit: This test config is only using one snapshot because there is only one testScript in the array below.